crates/sui-core/src/signature_verifier.rs

```diff
@@ -264,9 +275,27 @@ impl SignatureVerifier {
         });
     }
 
+    /// Insert a JWK into the verifier state. Returns true if the kid of the JWK has not already
+    /// been inserted.
+    pub(crate) fn insert_oauth_jwk(&self, content: &OAuthProviderContent) -> bool {
+        let mut oauth_provider_jwk = self.oauth_provider_jwk.write();
+
+        if oauth_provider_jwk.contains_key(content.kid()) {
+            return false;
+        }
+
+        let kid = content.kid().to_string();
+        oauth_provider_jwk.insert(kid, content.clone());
+        true
+    }
+
     pub fn verify_tx(&self, signed_tx: &SenderSignedData) -> SuiResult {
         self.signed_data_cache
-            .is_verified(signed_tx.full_message_digest(), || signed_tx.verify(None))
+            .is_verified(signed_tx.full_message_digest(), || {
+                let oauth_provider_jwk = self.oauth_provider_jwk.read().clone();
+                let aux_data = VerifyParams::new(None, oauth_provider_jwk);
```

Comment:
What does this `None` mean? By looking at https://github.com/MystenLabs/sui/blob/main/crates/sui-types/src/zk_login_authenticator.rs#L105, I'm confused. 

Is `VerifyParams::new(None, Default::default())` an op for non-zk-login authenticator?